### ISSUE TITLE
Javadoc for `PathPatternParser.defaultInstance` is outdated

### ISSUE THREAD

**Issue by chipu** [spring-projects/spring-framework#30976]
**Affects:** 6.0.11

Current Javadoc for `PathPatternParser defaultInstance`:

```java
	/**
	 * Shared, read-only instance of {@code PathPatternParser}. Uses default settings:
	 * <ul>
	 * <li>{@code matchOptionalTrailingSeparator=true}
	 * <li>{@code caseSensitivetrue}
	 * <li>{@code pathOptions=PathContainer.Options.HTTP_PATH}
	 * </ul>
	 */
```

But `matchOptionalTrailingSeparator` is `false`, and `=` is missing in the next line: `caseSensitive=true`

**Comment by sbrannen** [comment]
I've edited your comment to improve the formatting. You might want to check out this [Mastering Markdown guide](https://guides.github.com/features/mastering-markdown/) for future reference.

**Comment by sbrannen** [comment]
@chipu, congratulations on submitting your first issue for the Spring Framework! 👍 

This has been addressed in `6.0.x` and `main`.
